apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -160,6 +160,7 @@ def get_objects(element, data):
 def render(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     answer_name = pl.get_string_attrib(element, "answer-name")
```

Comment:
😬 `answer-name` vs `answers-name`, which means that the raised exception will have an incorrect error message.

(FWIW, I actually rather dislike `answers-name` as an attribute name, though we're in _much_ too deep to do anything about that now lmao)

Reply:
Ahhh, I didn't really think about this until you just mentioned it. Yeah, it's used in way too many places to really change. I'm also not sure what the status of this threejs element is, from some other issues it seems like there are a bunch of feature requests, but people on those have said this thing isn't really being maintained or used in any substantial way.